odkx-src/survey-sample-app-overview.rst

```diff
@@ -25,9 +25,9 @@ We have provided a sample application to help you acquaint yourself with the var
     - a form used to gather information about a household. 
     - No
     - | To operate correctly, this requires the *Household Member Survey* sub-form and the *Education* sub-form 
-      | (you should not open those sub-forms directly they are launched from within Household Survey).
+      | (you should not open those sub-forms directly, they are launched from within Household Survey).
   * - | **Select Examples** 
-    - | a form with several examples of select widgets, including widgets 
+    - | a form with several examples of selected widgets, including widgets 
```

Comment:
Here, I believe "select" is being used as a type of prompt. So it's not selected widgets. Could change from widgets to select prompts if that is clearer.

Reply:
> Here, I believe "select" is being used as a type of prompt. So it's not selected widgets. Could change from widgets to select prompts if that is clearer.

If I get you right, I can change the "widgets" to "prompts" ?

Reply:
Yes, double check in the select examples form if they are all select prompts, but if so, can change to select prompts rather than select widgets.

Reply:
> Yes, double check in the select examples form if they are all select prompts, but if so, can change to select prompts rather than select widgets.

Okay, I will check that. Thank you